src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -780,6 +780,7 @@ internal static bool IsZeroElementTupleType(TypeSymbol type)
                 bool isError = hasErrors || outPlaceholders.IsDefaultOrEmpty || i >= outPlaceholders.Length;
                 TypeSymbol elementType = isError ? CreateErrorType() : outPlaceholders[i].Type;
                 ParameterSymbol? parameter = null;
+                // PROTOTYPE(extended-property-patterns)
```

Comment:
Would be useful for PROTOTYPE markers to have a comment to clarify what needs to be handled (here and elsewhere)